keyboards/tweetydabird/lotus58/elite_c/rules.mk

```diff
@@ -0,0 +1 @@
+BOOTLOADER = atmel-dfu
```

Comment:
Should have a `info.json` instead of `rules.mk`, on this same folder, to setup the bootloader

Reply:
Sorry but these suggestions are completely incorrect, as then it does not become a build target.